src/views/thread/components/stickyHeader.js

```diff
@@ -52,6 +53,15 @@ const StickyHeader = (props: Props) => {
 
       {channel.channelPermissions.isMember && (
         <StickyHeaderActionsContainer>
+          <ActionsDropdown
+            thread={thread}
+            // toggleEdit={this.props.toggleEdit}
```

Comment:
Why are all of these commented out?

Reply:
Because we cannot toggle editing from the sticky header—it's also a strange experience as you don't even see the thread ("What am I editing?!").

Going to remove the comments!